main.tf

```diff
@@ -56,9 +56,18 @@ data "aws_iam_policy_document" "sns_kms_key_policy" {
   }
 }
 
+module "aws_sns_topic_label" {
+  source  = "cloudposse/label/null"
+  version = "0.24.1"
+  count   = local.enabled ? 1 : 0
```

Comment:
Isn't this already covered by the `context` ?

Reply:
Also I do not believe `local.enabled` is defined.

Reply:
it was, but removed it anyways